digital_twins/o5g/lifecycle/execute

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
I really like tmux but @mortenhaahr has experienced problems with using it. Is there a way to run this example without tmux? 
If there is an easier way to do it. We can add it as an option.

Reply:
@mortenhaahr can you provide more info on what problems you've had with tmux? It might be possible to supply a custom tmux config with the example to solve this.